### PR TITLE
Post restructure

### PR DESCRIPTION
**Pull request by Pixelguymm**

### CODE REVIEW

**Review by TimothyGillespie — APPROVED** [review]
Makes sense to me and works.
I could imagine a post fitting into mulitple categories - if that's the case one could make it a multi select and not require any category to be selected - that way "None" would be implicity contained as well.

**Comment by Pixelguymm** [comment]
Agreed, I think we can expand on the categories and maybe turn them into tags at some point along the way.

File: static/admin/config.yml [hunk 1]
```diff
@@ -16,11 +16,20 @@ collections:
...
+        hint: This will determine what label will show above the post title.
+        widget: select
+        options:
+          - { label: 'None', value: '' }
+          - { label: 'Announcement', value: 'announcement' }
           - { label: 'News', value: 'news' }
-          - { label: 'Music Pack', value: 'music-packs' }
-          - { label: 'Community Event', value: 'community-events' }
-          - { label: 'Map of the Week', value: 'maps-of-the-week' }
+          - { label: 'Guide', value: 'articles' }
```
Comment (TimothyGillespie) [inline static/admin/config.yml:30]: Sorry, this slipped past me. This seems odd to me that the value for `Guide` would be `articles` (also mismatch of plural/singular.

Comment (Pixelguymm) [reply static/admin/config.yml]: You're totally right! Fixed.

**Pull request merged**